components/brave_rewards/resources/tip/components/siteBanner.tsx

```diff
@@ -210,9 +216,9 @@ class Banner extends React.Component<Props, State> {
         addFundsLink={this.addFundsLink}
       >
       {
-        this.props.tweetMetaData
+        this.props.mediaMetaData && this.props.mediaMetaData.mediaType === 'twitter'
```

Comment:
let's check first for `!this.props.mediaMetaData` so that we don't need to check for every platform

Reply:
done